doctr/__main__.py

```diff
@@ -388,6 +402,74 @@ def configure(args, parser):
     in your .travis.yml.
     """.format(encrypted_variable=encrypted_variable.decode('utf-8'), N=N)))
 
+def enable_fork_build(args, parser):
+    if not args.force and on_travis():
+        parser.error("doctr appears to be running on Travis. Use "
+            "doctr configure --force to run anyway.")
+
+    if args.upload_key:
+        login_kwargs = GitHub_login()
+    else:
+        login_kwargs = {'auth': None, 'headers': None}
+
+    build_repo = input("What repo do you want to build the docs for (<yourname>/reponame, like 'drdoctr/doctr')? ")
+    is_private = check_repo_exists(build_repo, service='github', **login_kwargs)
+    check_repo_exists(build_repo, service='travis')
+
+    deploy_repo = input("What repo do you want to deploy the docs to? [{build_repo}/<branchname>] ".format(build_repo=build_repo))
+    if not deploy_repo:
+        deploy_repo = build_repo
+
+    if deploy_repo != build_repo:
+        check_repo_exists(deploy_repo, service='github', **login_kwargs)
+
+    N = IncrementingInt(1)
+
+    header = "\n================== You should now do the following ==================\n"
+
+    if args.token:
```

Comment:
Yes, thanks,  I'll clean things up, I'm just prototyping by copy-pasting for now.